guide/payments/privacy.md

```diff
@@ -0,0 +1,137 @@
+- Single-use addresses (avoiding address reuse)
+- Coin selection / labelling
+- Coin join / Pay join
+-->
+
+It’s a common misconception that Bitcoin is anonymous. Instead, it is referred to as pseudonymous which means each address you generate is like a new identity. Unless you reveal that you are the owner whether directly or indirectly you are able to remain anonymous.
+
+When mined, transactions, their signatures, and addresses are public on the ledger, forever. There is no personally identifiable information attached to transactions on the blockchain though. The transparent property of the blockchain allows anyone to easily verify the amount of Bitcoin in existence but also makes looking up any address or transaction trivial. This can be demonstrated going back to the first block on [2009-01-03](https://blockstream.info/tx/4a5e1e4baab89f3a32518a88c31bc87f618f76673e2cc77ab2127b7afdeda33b), a transaction mined by Satoshi Nakamoto.
+
+> Each Bitcoin transaction contains at least one input and at least one output. This means that once a single address is known, there is a trail to follow the Bitcoin.
+> 
+> <cite>As documented by <a href="https://docs.wasabiwallet.io/FAQ/FAQ-GeneralBitcoinPrivacy.html#how-is-bitcoin-bad-in-terms-of-privacy">Wasabi Wallet</a></cite>
+
+The key to keeping your transactions private is to prevent others from determining which addresses you own[^3]. Satoshi let others know that they mined the first block, which contained just one transaction, the block reward sent to their address. This illustrates the permanence of the association to an address and an identity. That being said, the pseudonym “Satoshi Nakamoto” also contributes to further anonymity. Once an address owner's identity is made public it becomes a difficult task to break the connection.
+
+<!-- TODO: add graphic and heading that demonstrate points of compromise when transacting with bitcoin -->
+
+## Methods to preserve privacy
+
+Keeping private takes constant and diligent work but is not impossible to achieve an acceptable level of privacy[^1]. Let’s explore some methods and practices that can help preserve privacy on a transaction level.
+
+<!-- talk about the problem as you are talking about the solution -->
+
+### Generate a new address for each payment
+
+Creating new online personas is not the only way to preserve privacy. A new address can be generated by the wallet application to receive bitcoin. This is done using [HD Wallets]({{ "/guide/glossary/#hd-wallet" | relative_url }}), a standard for how private keys get generated in modern bitcoin applications. This allows each transaction not to be tied to a single address, making it more difficult to make an association to the owner.
+
+> If thieves can see your income, holdings, and spending, they can use this information to target and exploit you.[^2]
```

Comment:
No need for a footnote, you can use the blockquote author field for the link.

Reply:
Fixed in https://github.com/BitcoinDesign/Guide/pull/349/commits/d9b82106d5ec388b3068fbe7bb73f452db4d3249